lms/static/js/utils/dropdown.js

```diff
@@ -0,0 +1,235 @@
+var edx = edx || {},
```

Comment:
I'd suggest you look at existing tests to see how to start one-- for example, /common/lib/xmodule/xmodule/js/spec/video/video_accessible_menu_spec.js (which you modified) tests the same sort of things you will want to test. /edx-platform/docs/en_us/internal/testing.rst explains how to run Jasmine tests. Give it a try, and I'm happy to help with questions.